notebooks/Data cleaning.ipynb

```diff
@@ -0,0 +1,713 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 76,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "'''\n",
+    "Basic data cleaning notebook\n",
+    "'''\n",
+    "import sqlite3\n",
+    "import pandas as pd\n",
+    "import sys\n",
+    "if '..' not in sys.path:\n",
+    "    sys.path.insert(0, '..')\n",
+    "from src.loaders.config import Config\n",
+    "\n",
+    "# Create config object\n",
+    "config = Config('../config')\n",
+    "\n",
+    "# make a sql connection\n",
+    "# use temporary db\n",
+    "config.database = '../data/boardgames.sqlite3'\n",
```

Comment:
This is should be `'../data/final/boardgames.sqlite3'`, as that is where loader should be saving the sqlite file. PR #20 adds the config.DB_NAME, and config.DB_PATH params as well, which would prevent this issue in the first place.